snow/engine/avalanche/bootstrap/bootstrapper.go

```diff
@@ -426,14 +427,12 @@ func (b *bootstrapper) process(ctx context.Context, vtxs ...avalanche.Vertex) er
 	vtxHeightSet := set.Set[ids.ID]{}
 	prevHeight := uint64(0)
 
-	for toProcess.Len() > 0 { // While there are unprocessed vertices
+	for toProcess.Len() > 0 {
 		if b.Halted() {
 			return nil
 		}
 
-		vtx := toProcess.Pop() // Get an unknown vertex or one furthest down the DAG
-		vtxID := vtx.ID()
-
+		vtxID, vtx, _ := toProcess.Pop()
```

Comment:
> we skip the boolean since we are sure we can pop an existing element (heap length is non zero). I wonnder if we should try and do the pop in the for loop directly?

Yeah popping in the for-loop makes more sense here imo.